src/style.css

```diff
@@ -1 +1,31 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+}
```

Comment:
Don't use the `*` selector (it impacts performance). Set styles only for elements that require them.